### PR TITLE
add readme doc

### PR DESCRIPTION
**Pull request by bilalelbedoui93**
he añadido un readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# HELLO WORLD 
+
+## Intro
```
Comment (daniscript) [inline README.md:3]: @bilalelbedoui93 Intro no, introduction yes

File: README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+Explain project
+
+## Function Description
+
+To do describe functionality
```
Comment (daniscript) [inline README.md]: @bilalelbedoui93 hei, you should finish this work

**Review by daniscript — COMMENTED** [review]
Check commets

File: README.md (hunk 1 above)
Comment (bilalelbedoui93) [reply README.md:3]: @DaniLatreCodes sorry, but "intro" is ok

File: README.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+# HELLO WORLD 
+
+## Intro
+
+Explain project
+
+## Function Description
+
+Lorem ipsum ...
```
Comment (bilalelbedoui93) [inline README.md:9]: @DaniLatreCodes  it's not enough, there are some lines missing

**Pull request merged**